packages/venia-ui/lib/components/FilterModal/FilterList/filterItem.js

```diff
@@ -19,20 +19,31 @@ const FilterItem = props => {
         [title, value]
     );
 
-    const handleClick = useCallback(() => {
-        toggleItem({ group, item });
+    const handleClick = useCallback(
+        e => {
+            if (e.button !== 0) return;
```

Comment:
Could we had comments where we have those conditions to explain why we do this?
Also, could we update the QA Steps to make sure we test this out?

Reply:
Yes sorry, I'll add some comments. The problem here is that by switching to onMouseDown events every click (right, left, middle) can trigger it, so I'm making sure we only call the action on left mouse click. This was an error from a bug fix in [PWA-1784](https://jira.corp.magento.com/browse/PWA-1784)